scheduler/src/cook/kubernetes/compute_cluster.clj

```diff
@@ -142,7 +94,7 @@
                         (let [nodes @current-nodes-atom
                               pods @current-pods-atom
                               offers (generate-offers nodes pods this)
-                              pool (config/default-pool)
+                              pool (or (config/default-pool) "no-pool")
```

Comment:
This seems like a layering violation. The pool should be defaulted at a higher level. Actually, it should be passed as an argument to the compute cluster factory method. (One pool can have multiple compute clusters.)

Reply:
This is deleted in a subsequent PR, can we just wait until then?

Reply:
Works for me.